### PR TITLE
Differ testing

### PR DESCRIPTION
**Pull request by LucPrestin**
- Adds interface tests for the differ class
- Fixes code so that the tests pass

### CODE REVIEW

File: lively.merger/tests/differ-test.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      }).to.throw('Cannot diff morphs, morph1 with id morph1id not found');
+    });
+
+    it('detects if the morphs with the second ID is not alive', () => {
+      expect(() => {
+        Differ.diffMorphsWithIds(morph1.id, 'morph2id');
+      }).to.throw('Cannot diff morphs, morph2 with id morph2id not found');
+    });
+
+    it('runs without error if both morphs are alive', () => {
```
Comment (T4rikA) [inline lively.merger/tests/differ-test.js:44]: I see why we would like to test this, but I have the feeling even if it doesn't throw an error it doesn't mean it works properly

Comment (T4rikA) [reply lively.merger/tests/differ-test.js:44]: Think we should check the result of the diff

Comment (LucPrestin) [reply lively.merger/tests/differ-test.js:44]: Hmm, fair. I tried to test the result edge-cases (like empty differingProperties if the object is the same), but really testing for the result is somewhat hard, as the function only works on morphs and their properties are really large... Any idea how to get around that?

Comment (T4rikA) [reply lively.merger/tests/differ-test.js:44]: I mean essentially we are testing diffMophs here, which we are testing below again.
Maybe we could just delete this test because it is not needed?

Comment (LucPrestin) [reply lively.merger/tests/differ-test.js:44]: fair enough

**Review by T4rikA — APPROVED** [review]
